doc/api/tls.md

```diff
@@ -1094,6 +1094,37 @@ See
 [SSL_get_shared_sigalgs](https://www.openssl.org/docs/man1.1.1/man3/SSL_get_shared_sigalgs.html)
 for more information.
 
+### `tlsSocket.exportKeyingMaterial(length, label)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `length` {number}
+  number of bytes to retrieve from keying material
+* `label` {string} an application specific label, typically this will be a
+value from the IANA Exporter Label Registry
+(https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#exporter-labels).
+
+* Returns: {Buffer} {number} of bytes of the keying material
```

Comment:
```suggestion
* Returns: {Buffer} number of bytes of the keying material
```

Reply:
Actually, if this is a `Buffer` then the description doesn't quite make sense

Reply:
changed the description to * Returns: {Buffer} requested bytes of the keying material